### DISCUSSION TITLE
Scheme for Max 0.4 beta 1  (Mac and Windows)

### DISCUSSION THREAD

**Discussion by iainctduncan** [iainctduncan/scheme-for-max#137] [General]
Scheme for Max 0.4.0 for Mac (Intel and M1), Win 64.

Tested on Max 8 and Max for Live, on Windows 10 and Mac up to 12.1 (Monterey)
Supports Apple Silicon. Note: win32 is no longer provided (let me know if that is an issue please).

Instructions: download the zip file and expand it in your Max Packages directory. This should enable you to find it in the package manager and launch the help file. You may need to do the usual security authorization rituals (see readme on the github page).

If you hit issues with installation please feel free to post in the discussion on the forum or make github issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/iainctduncan/scheme-for-max/releases/tag/0.4.0-rc1'>Scheme for Max 0.4 beta 1  (Mac and Windows)</a>.</em>

**Comment by DerKastellan** [comment]
I know it's a bit of a hype-train kind of question, but do you know if s4m can also be used within the context of RNBO?
